tests/test_datetime.py

```diff
@@ -1,6 +1,25 @@
 import unittest
+from datetime import datetime, timezone
+
+from hypothesis import given, settings, strategies as st
+from hypothesis.extra.dateutil import timezones as dateutil_timezones
+
+TIME_ZONES_STRATEGY = st.one_of(
+    st.sampled_from([None, timezone.utc]), dateutil_timezones()
+)
 
 
 class TestDatetime(unittest.TestCase):
     # TODO: https://docs.python.org/3/library/datetime.html
-    pass
+
+    # The search space here is large and as of January 2020 the sampling
+    # functions for characters() and datetimes() are not biased towards
+    # "interesting" examples, so it's worth throwing some extra cycles at this.
+    @settings(max_examples=2500)
```

Comment:
Instead of bumping up the number of examples, I've modified the `sep` strategy to be "any character, or any surrogate character" to bias the chances.

Main motivation is to ensure that it's easy to globally change the number of examples with a profile, and keep our options open re: continuous fuzzing.  There's no tooling for that yet, but converting Hypothesis tests to fuzz targets is on my wishlist and conceptually simple ("just a matter of code")...